pages/clustering/high-availability.mdx

```diff
@@ -463,70 +463,91 @@ You can directly use initialization file `HA_init.cypherl`:
-    network_mode: host
+      - MEMGRAPH_HA_CLUSTER_INIT_QUERIES=/tmp/init/HA_register.cypher
+    command: [ "--init-file=/tmp/init/license.cypher", "--log-level=TRACE", "--data-directory=/tmp/mg_data_coord1", "--log-file=/tmp/coord1.log", "--also-log-to-stderr", "--coordinator-id=1", "--coordinator-port=10111", "--experimental-enabled=high-availability"]
+    networks:
+      memgraph_ha:
+        ipv4_address: 172.21.0.4
 
   coord2:
-    image: "memgraph/memgraph-mage"
+    image: "memgraph/memgraph"
     container_name: coord2
     volumes:
-      - ./license.cypherl:/tmp/init/license.cypherl:ro
-    command: ["--bolt-port=7691", "--init-file=/tmp/init/license.cypherl", "--log-level=TRACE", "--data-directory=/tmp/mg_data_coord2", "--log-file=/tmp/coord2.log", "--coordinator-id=2", "--coordinator-port=10112", "--experimental-enabled=high-availability"]
-    network_mode: host
+      - ./license.cypher:/tmp/init/license.cypher:ro
+    command: [ "--init-file=/tmp/init/license.cypher", "--log-level=TRACE", "--data-directory=/tmp/mg_data_coord2", "--log-file=/tmp/coord2.log", "--also-log-to-stderr", "--coordinator-id=2", "--coordinator-port=10112", "--experimental-enabled=high-availability"]
+    networks:
+      memgraph_ha:
+        ipv4_address: 172.21.0.2
 
   coord3:
-    image: "memgraph/memgraph-mage"
+    image: "memgraph/memgraph"
     container_name: coord3
     volumes:
-      - ./license.cypherl:/tmp/init/license.cypherl:ro
-    command: ["--bolt-port=7692", "--init-file=/tmp/init/license.cypherl", "--log-level=TRACE", "--data-directory=/tmp/mg_data_coord3", "--log-file=/tmp/coord3.log", "--coordinator-id=3", "--coordinator-port=10113", "--experimental-enabled=high-availability"]
-    network_mode: host
-    
+      - ./license.cypher:/tmp/init/license.cypher:ro
+    command: [ "--init-file=/tmp/init/license.cypher",  "--log-level=TRACE", "--data-directory=/tmp/mg_data_coord3", "--log-file=/tmp/coord3.log", "--also-log-to-stderr", "--coordinator-id=3", "--coordinator-port=10113", "--experimental-enabled=high-availability"]
+    networks:
+      memgraph_ha:
+        ipv4_address: 172.21.0.3
+
   instance1:
-    image: "memgraph/memgraph-mage"
+    image: "memgraph/memgraph"
     container_name: instance1
     volumes:
-      - ./license.cypherl:/tmp/init/license.cypherl:ro
-    command: ["--bolt-port=7687", "--init-file=/tmp/init/license.cypherl","--data-recovery-on-startup=true", "--log-level=TRACE", "--data-directory=/tmp/mg_data_instance1", "--log-file=/tmp/instance1.log", "--management-port=10011", "--experimental-enabled=high-availability"]
-    network_mode: host
-    
+      - ./license.cypher:/tmp/init/license.cypher:ro
+    command: ["--init-file=/tmp/init/license.cypher","--data-recovery-on-startup=true", "--log-level=TRACE", "--data-directory=/tmp/mg_data_instance1", "--log-file=/tmp/instance1.log", "--also-log-to-stderr", "--management-port=10011", "--experimental-enabled=high-availability"]
```

Comment:
I think --data-recovery-on-startup is by default set to true so there is no need to add it (please check this).  Applies to other services as well.

Reply:
it is still false, nobody changed it yet

Reply:
In docs -> --data-recovery-on-startup=true Source: https://memgraph.com/docs/database-management/configuration#other

cc @kgolubic